src/joining-the-team.rst

```diff
@@ -0,0 +1,35 @@
+Joining the team!
+*****************
+
+``conda-forge`` is a volunteer-driven community. We need your help! If
+you are interested in helping us maintain this ecosystem, our biggest
+need is help reviewing new conda packages coming in through staged-recipes.
+If you would like to help us do this please ask one of the core members on
+`our gitter channel <https://gitter.im/conda-forge/conda-forge.github.io>`__.
+We will then reach consensus in private and let you know our decision.
+
+Our decision will likely be "yes" but it could be "please get more involved
+with the review process on staged recipes first" if we have not seen you
+take much interest in the staged-recipes review process.
+
+For transparency, our process is to have the core team "vote" on adding new
+members. When we have a new member candidate, we ask the core team to vote
+yes/no and give the team 24h to vote. The person will then be given merge
+rights to conda-forge/staged-recipes so that they can help us review and merge
+submissions more quickly!
+
+staged-recipes responsibilities
+===============================
+You are the welcoming committee for new recipes coming in to the conda-forge
+community! Please give new (and experienced) contributors a pleasant experience!
+Generally speaking your role is as follows:
+
+1. Keep up to date with the current best practices for conda packaging standards
```

Comment:
We might consider putting out a changelog of best practices somewhere so people can be up to date.

Reply:
There's [the guidelines]( https://conda-forge.org/docs/guidelines.html ). We could also start versioning the docs. They have a version, but haven't tried to come up with a scheme yet.

Suggestions welcome. Feel free to put this in a new issue/PR of your choosing. :)